File: CarzineCore/Models/PreOrderModel.cs
﻿using Newtonsoft.Json;

namespace CarzineCore.Models
{
	public class PreOrderModel
	{
		public int Id { get; set; }

		public string Phone { get; set; }

		public string UserEmail { get; set; }

		public StandardProductModel Product { get; set; }

		public DateTime Date { get; set; }

		public int ClientStatus { get; set; }

		public SupplyerStatus SupplyerStatus { get; set; }

		public string PaymentOrderState { get; set; }
	}
}

File: Carzine/ClientApp/src/app/models/PreOrderModel.ts
export class OrderModel {
    id: number | undefined;
    phone: string | undefined;
    user_email: string | undefined;
    date: Date | undefined;
    client_status_id: number | undefined;
    clientStatusName: string = '';
    payment_Order_State: string = '';
    total_Sum: number = 0;
}